Makefile

```diff
@@ -21,23 +21,27 @@ all: pkg tar
 pkg: deb rpm
 
 deb: populate-deb
+	(cd pkg/deb/; \
+		DEBMAIL="stackdriver-agents@google.com" DEBFULLNAME="Stackdriver Agents" \
+		dch --package "${PACKAGE_NAME}" \
+		-v "${PACKAGE_VERSION}" "Automated Build")
 	cp -a pkg/deb/debian ${DEB_PACKAGE_DIR}
 	(cd ${DEB_PACKAGE_DIR} && debuild --no-tgz-check -us -uc)
 
 rpm: populate-el
-	rpmbuild -v -bb --nodeps --target noarch --define "buildroot `pwd`/${RPM_PACKAGE_DIR}/files" --define "_rpmdir `pwd`/${BUILD_DIR}/el" --define "package_version ${PACKAGE_VERSION}" --define "package_build_num 1" pkg/el/google-fluentd-catch-all-config.spec
+	rpmbuild -v -bb --nodeps --target noarch --define "buildroot `pwd`/${RPM_PACKAGE_DIR}/files" --define "_rpmdir `pwd`/${BUILD_DIR}/el" --define "package_version ${PACKAGE_VERSION}" --define "package_build_num 1" "pkg/el/${PACKAGE_NAME}.spec"
 
 tar: deb-tar el-tar
 
 # tarfile for Debian systems
 deb-tar: populate-deb
 	tar --owner=root --group=root -C ${DEB_FILES_DIR} \
-	    -czf ${PACKAGE_NAME}_${PACKAGE_VERSION}.deb.tar.gz .
+			-czf ${PACKAGE_NAME}_${PACKAGE_VERSION}.deb.tar.gz .
```

Comment:
I switched everything to tabs, as the majority of indentations were tabs previously, with a few outliers.

Reply:
Please revert this. `make` requires all commands to start with a (single) tab, but within that command, the indentation should be spaces.

Reply:
Eek, done.